README.md

```diff
@@ -10,99 +10,102 @@
 
-With Mirage the objects in the client are mirror images of the objects in the server.  Mirage provides all the tools necessary to keep them in sync and pass messages between them.
+Mirage is built by passionate network engineers and is backed by a friendly community.
 
 ## Installation
 
 To install Mirage, follow these steps:
 
-1) Install [Unity 2020.1.17 or later](https://unity.com/)
+1) Install [Unity 2020 LTS or later](https://unity.com/) - _LTS versions are strongly recommended._
 2) Start a new project or open your existing one
-3) Add openupm registry.  Click on the menu Edit -> Project settings...,  and add a scoped registry like so: <br/>
+3) Add the OpenUPM registry.  Click on the `Edit` menu, then select `Project settings...`, select `Package Manager` and add a scoped registry like so: <br/>
     Name: `OpenUPM` <br/>
     Url: `https://package.openupm.com` <br/>
     Scopes:
     - `com.cysharp.unitask`
     - `com.openupm`
     - `com.miragenet`
    ![Scoped Registry](doc/images/Scoped%20Registry.png)
-4) Close the project settings
-5) Open the package manager.  Click on menu Window -> Package Manager and select "Packages: My Registries", select the latest version of Mirage and click install, like so:
+4) Close the project settings.
+5) Open the package manager by clicking on the `Window` menu and selecting `Package Manager`. Then select `Packages`, `My Registries`, select the latest version of Mirage and click install, like so:
    ![Install Mirage](doc/images/Install%20Mirage.png)
-6) You may come back to the package manager to unistall Mirage or upgrade it.
+6) You may come back to the package manager at any time to uninstall Mirage or upgrade it.
 
 ## Comparison with Mirror
-When migrating a project from Mirror to Mirage, these will be the most notable differences.
+
+Mirage has some notable differences from its distant sister, Mirror. The table below briefly details them:
 
 | Mirage                                              | Mirror                                 |
 | --------------------------------------------------- | -------------------------------------- |
-| Install via Unity Package Manager                   | Install from Asset Store               |
+| Installs via Unity Package Manager                  | Installs from Asset Store              |
 | Errors are thrown as exceptions                     | Errors are logged                      |
 | `[ServerRpc]`                                       | `[Command]`                            |
-| `[ClientRpc(target=Client.Owner)]`                  | `[TargetRpc]`                          |
+| `[ClientRpc(target = Client.Owner)]`                | `[TargetRpc]`                          |
```

Comment:
I believe the `Client.Owner` has been renamed to `RpcTarget.Owner`. Also maybe adding a link to our migration guide docs would be nice.

Reply:
Adjusted in the latest commit.